aspnet/signalr/overview/performance/using-signalr-performance-counters-in-an-azure-web-role.md

```diff
@@ -1,117 +1,143 @@
-- In **Solution Explorer**, right-click the **Startup** folder and select **Add...**, **Existing Item**. In the dialog that appears, select **SignalR.exe** and click **Add**.
+11. In **Solution Explorer**, right-click the *Startup* folder and select **Add &gt; Existing Item...**. In the dialog that appears, select *signalr.exe* and click **Add**.
 
-    ![Add SignalR.exe to project](using-signalr-performance-counters-in-an-azure-web-role/_static/image6.png)
-- Right-click on the **Startup** folder you created. Select **Add**, **New Item**. Select the **General** node, select **Text File**, name the new item `SignalRPerfCounterInstall.cmd`. This command file will install the SignalR performance counters to the web role.
+    ![Add signalr.exe to project](using-signalr-performance-counters-in-an-azure-web-role/_static/image6.png)
+    
+12. Right-click on the *Startup* folder you created. Select **Add &gt; New Item...**. Select the **General** node, select **Text File**, and name the new item *SignalRPerfCounterInstall.cmd*. This command file will install the SignalR performance counters into the web role.
 
     ![Create SignalR performance counter installation batch file](using-signalr-performance-counters-in-an-azure-web-role/_static/image7.png)
-- Right-click the **SignalR.exe** file, and select **Properties**. Set **Copy to Output Directory** to **Copy Always**.
+     
+13. When Visual Studio creates the *SignalRPerfCounterInstall.cmd* file, it will automatically open in the main window. Replace the contents of the file with the following script, then save and close the file. This script executes *signalr.exe*, which adds the SignalR performance counters to the role instance.
+
+    [!code-console[Main](using-signalr-performance-counters-in-an-azure-web-role/samples/sample3.cmd)]
+    
+14. Click the *signalr.exe* file in **Solution Explorer**. In the file's **Properties**, set **Copy to Output Directory** to **Copy Always**.
 
     ![Set Copy to Output Directory to Copy Always](using-signalr-performance-counters-in-an-azure-web-role/_static/image8.png)
-- Repeat the previous step for the **SignalRPerfCounterInstall.cmd** file.
-- Open **SignalRPerfCounterInstall.cmd** and enter the following script, then save and close the file. This script executes SignalR.exe, which adds the SignalR performance counters to the role instance.
+    
+15. Repeat the previous step for the *SignalRPerfCounterInstall.cmd* file.
 
-    [!code-console[Main](using-signalr-performance-counters-in-an-azure-web-role/samples/sample3.cmd)]
-- Right-click on the **SignalRPerfCounterInstall.cmd** file and select **Open With...** In the dialog that appears, select **Binary Editor** and click **OK**.
+    
+16. Right-click on the *SignalRPerfCounterInstall.cmd* file and select **Open With...**. In the dialog that appears, select **Binary Editor** and click **OK**.
```

Comment:
I didn't change elsewhere in here (e.g., **Add &gt; Existing Item...**). It seems to me that its best to reflect exactly what the UI shows. However, if you want it removed here, should I remove it everywhere in the doc?